src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -7,44 +7,91 @@
 
 package frc.robot.subsystems;
 
-import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 import com.revrobotics.CANPIDController;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel;
 import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMax.IdleMode;
 
 import frc.robot.Constants;
 
 public class ShooterSubsystem extends SubsystemBase {
-  private final CANSparkMax ShooterMAX = new CANSparkMax(Constants.SPARKMAX_SHOOTER_CAN_ADDR,
+  /**
+   * Initializes the SparkMAX motor controller, assigns it to the CAN address
+   * specified, and sets it to the NEO Brushless Motor.
+   */
+  private final CANSparkMax shooterMAX = new CANSparkMax(Constants.SPARKMAX_SHOOTER_CAN_ADDR,
       CANSparkMaxLowLevel.MotorType.kBrushless);
-
-  private final CANPIDController ShooterPID = new CANPIDController(ShooterMAX);
-
-  private final Encoder shootEncoder = new Encoder(Constants.SHOOTER_ENCODER_CHAN_A, Constants.SHOOTER_ENCODER_CHAN_B);
-
-  private final double ShooterSetpoint = 1.0;
-
+  /**
+   * The built-in PID controller provided by the Spark MAX motor controller.
+   */
+  private final CANPIDController shooterPID = new CANPIDController(shooterMAX);
+  /**
+   * The target velocity of the NEO Brushless Motor.
+   */
+  private final double shooterSetpoint = 1.0;
```

Comment:
Expose this setpoint so it can be controlled.  A good first step is using MoPrefs.  If it needs to change too much, then break out as a Shuffleboard control.
It might also depend on the range calculation from the LimeLight.